ui/app/AppLayouts/Wallet/popups/simpleSend/SimpleSendModal.qml

```diff
@@ -189,6 +189,14 @@ StatusDialog {
     signal formChanged()
     /** Output signal to launch buy flow **/
     signal launchBuyFlow()
+    /** Input signal called after all data is filled **/
+    signal afterOpened()
+
+    onAfterOpened: {
```

Comment:
why does this look like a hot fix? can we address it the right way on master? @Cuteivist

Reply:
off course! I just feel this way of adding new properties and signal should not be propagated into master that all, of thats case, I'll approve this PR :)